src/analysis/cytoscape/cytoscape.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import PurePath
```

Comment:
Let' s move this file up one level so the path is `src/analysis/cytoscape.py`  Otherwise it will be the only file in the `cytoscape` subdirectory.

Reply:
done